examples/user_guide/Network_Graphs.ipynb

```diff
@@ -340,7 +340,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Alternatively we can select the nodes and edges by an attribute that resides on either. In this case we will select the nodes and edges for a particular circle and then overlay just the selected part of the graph on the datashaded plot. In this way a smaller subgraph can be highlighted and the larger graph can be datashaded."
+    "Alternatively we can select the nodes and edges by an attribute that resides on either. In this case we will select the nodes and edges for a particular circle and then overlay just the selected part of the graph on the datashaded plot. Note that selections on the ``Graph`` itself will select all nodes that connect to one of the selected nodes. In this way a smaller subgraph can be highlighted and the larger graph can be datashaded."
```

Comment:
Is that a policy that can be selected?  Seems likely that's what people want, but not always.

Reply:
Not currently no, but I guess I could. I do provide an alternative way of doing it by selecting on the source and target instead. Any suggestion on what such an option could be called, don't like my suggestions much: ``only_selected`` or ``include_connected``.

Reply:
``include_targets``?

Reply:
Doesn't seem correct, the graph might be directed in which case that's just confusing because you may actually be selecting the source.

Reply:
I'm going to go with ``selection_mode='nodes'/'edges'`` unless there is a better suggestion.

Reply:
Ok by me.